src/libs/actions/IOU.js

```diff
@@ -76,13 +76,16 @@ function requestMoney(report, amount, currency, payeeEmail, participant, comment
     if (chatReport.iouReportID) {
         if (isPolicyExpenseChat) {
             moneyRequestReport = {...iouReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReport.iouReportID}`]};
-            moneyRequestReport.total += amount;
+
+            // Because of the Expense reports are stored as negative values, we substract the total from the amount
+            moneyRequestReport.total = ReportUtils.isExpenseReport(moneyRequestReport) ? moneyRequestReport.total - amount : moneyRequestReport.total + amount;
         } else {
             moneyRequestReport = IOUUtils.updateIOUOwnerAndTotal(iouReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReport.iouReportID}`], payeeEmail, amount, currency);
         }
     } else {
         moneyRequestReport = isPolicyExpenseChat
-            ? ReportUtils.buildOptimisticExpenseReport(chatReport.reportID, chatReport.policyID, payeeEmail, amount, currency)
+            ? // The amount for Expense reports are stored as negative value in the database
+              ReportUtils.buildOptimisticExpenseReport(chatReport.reportID, chatReport.policyID, payeeEmail, amount * -1, currency)
```

Comment:
Can we just add the `amount * -1` logic to the `buildOptimisticExpenseReport` method? That way we can also get rid of the comment here